azure-maven-plugin-lib/src/main/java/com/microsoft/azure/toolkit/maven/common/messager/AzureMavenMessager.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.maven.common.messager;
+
+import com.microsoft.azure.toolkit.lib.common.messager.IAzureMessager;
+import com.microsoft.azure.toolkit.lib.common.utils.TextUtils;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.annotation.Nonnull;
+import java.util.Objects;
+
+@Slf4j
+public class AzureMavenMessager implements IAzureMessager {
+    @Override
+    public void success(@Nonnull String message, @Nonnull String... title) {
+        log.info(message);
+    }
+
+    @Override
+    public void info(@Nonnull String message, @Nonnull String... title) {
+        log.info(message);
+    }
+
+    @Override
+    public void warning(@Nonnull String message, @Nonnull String... title) {
+        log.warn(message);
+    }
+
+    @Override
+    public void error(@Nonnull String message, @Nonnull String... title) {
+        log.error(message);
+    }
+
+    @Override
+    public void error(@Nonnull Throwable throwable, String... title) {
+        final String message = Objects.nonNull(title) ? title[0] : throwable.getMessage();
```

Comment:
title[0] is not safe,  will throw IndexOutOfBoundsException

Reply:
> title[0] is not safe, will throw IndexOutOfBoundsException

can we pass a `new String[0]` to `String...` varargs?